src/test/java/com/wikia/webdriver/testcases/featuredVideo/FeaturedVideoDesktopTests.java

```diff
@@ -0,0 +1,130 @@
+package com.wikia.webdriver.testcases.featuredVideo;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.featuredvideo.FeaturedVideoComponentObject;
+
+import org.testng.annotations.Test;
+
+@Test(groups = {"FeaturedVideoDesktop"})
+@Execute(onWikia = "featuredvideo", asUser = User.ANONYMOUS)
+public class FeaturedVideoDesktopTests extends NewTestTemplate {
+
+  @Test
+  public void videoIsPresentOnArticle() {
+    FeaturedVideoComponentObject video = new FeaturedVideoComponentObject()
+        .setAutoplayCookie(false)
+        .openWikiArticle("FeaturedVideo");
+
+    Assertion.assertTrue(video.isFeaturedVideo());
+  }
+
+  @Test
+  public void videoTitleIsVisible() {
+    FeaturedVideoComponentObject video = new FeaturedVideoComponentObject()
+        .setAutoplayCookie(false)
+        .openWikiArticle("FeaturedVideo");
+
+    Assertion.assertEquals(video.getTitle(), "Why Jon Snow Should Rule Westeros");
```

Comment:
can we create our own video in jw dashboard with our custom titles, captions and so on? If I get it currently the one used in tests is just some production video copied to our article. If somebody from content team changes something with this video our tests will fail (even if feature itself works)

Reply:
done!